src/modules/help/i18n/firstSteps.provider.js

```diff
@@ -0,0 +1,29 @@
+export const firstStepsProvide = (lang) => {
+	switch (lang) {
+
+		case 'en':
+			return {
+				//the first part of the snake_case key should be the name of the related module
```

Comment:
Probably we will have more components within the help module in the future. Let's give them keys indicating their
concrete component:
help_button -> help_first-steps_button